README.md

```diff
@@ -13,18 +13,32 @@ A lightweight web interface to manage emails sent through [LocalStack SES](https
 - **Download Emails:** Save email contents in EML format for offline access or further analysis.
 
 ## Usage
-
+### With Node
 1. **Clone and Install:**
    - Clone the repository: `git clone https://github.com/veertech/localstack-aws-ses-email-viewer.git`
    - Navigate to the project directory: `cd localstack-aws-ses-email-viewer`
    - Install dependencies: `npm install`
 
 2. **Run the Application:**
+   - Run: `export LOCALSTACK_HOST=http://localhost:4566`
+   - Run: `export PORT=3005`
    - Run: `node server.js`
 
 3. **Access the Interface:**
    - Open [http://localhost:3005](http://localhost:3005) in your web browser.
 
+### With Docker
+1. **Clone and Install:**
+   - Clone the repository: `git clone https://github.com/veertech/localstack-aws-ses-email-viewer.git`
+   - Navigate to the project directory: `cd localstack-aws-ses-email-viewer`
```

Comment:
can we move this part above so it is not repeated for both sections? 

---

## Usage

- Clone the repository: `git clone https://github.com/veertech/localstack-aws-ses-email-viewer.git`
- Navigate to the project directory: `cd localstack-aws-ses-email-viewer`

### Run locally

- Install dependencies: `npm install`
- Run: `LOCALSTACK_HOST=http://localhost:4566 PORT=3005 node server.js`
- Open [http://localhost:3005](http://localhost:3005) in a web browser.

### Run with docker

- Build image: `docker build  . -t ses-viewer`
- Run: `docker run --rm --env LOCALSTACK_HOST=http://localstack:4566 -p 3005:3005 ses-viewer`
- Open [http://localhost:3005](http://localhost:3005) in a web browser.